<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetGlobalFormButtons</class>
 <widget class="QWidget" name="WidgetGlobalFormButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>43</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QPushButton" name="button_cancel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../data/ui_pictures.qrc">
       <normaloff>:/buttons/logo/images/cancel.png</normaloff>:/buttons/logo/images/cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_revert">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Revert modifications</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../data/ui_pictures.qrc">
       <normaloff>:/buttons/logo/images/revert.png</normaloff>:/buttons/logo/images/revert.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_ok">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Validate</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../data/ui_pictures.qrc">
       <normaloff>:/buttons/logo/images/apply.png</normaloff>:/buttons/logo/images/apply.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../data/ui_pictures.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>applyClicked()</signal>
  <signal>revertClicked()</signal>
  <signal>cancelClicked()</signal>
 </slots>
</ui>
